RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -1751,11 +1996,11 @@ class DeepTauId : public deep_tau::DeepTauBase {
                                                  false);
       get(dnn::pfCand_ele_pvAssociationQuality) =
           getValueLinear<int>(candFunc::getPvAssocationQuality(ele_cand), 0, 7, true);
-      get(dnn::pfCand_ele_puppiWeight) = getValue(candFunc::getPuppiWeight(ele_cand));
+      get(dnn::pfCand_ele_puppiWeight) = is_inner ? getValue(candFunc::getPuppiWeight(ele_cand, 0.9906834491656403))
```

Comment:
I noticed now that the second argument is actually supposed to be a float.
To avoid confusion due to double->float conversions I suggest to change to float literals here and in other cases.